lib/armbian-configng/config.ng.jobs.json

```diff
@@ -725,8 +725,120 @@
                     "src_reference": "",
                     "author": "",
                     "condition": "[ -f /usr/bin/armbianmonitor ]"
-        
-                }
+
+                },
+                {
+                    "id": "SW01.4",
+                    "description": "Softy - 3rd party apps",
+                    "sub": [
+                            {
+                                "id": "SW21",
+                                "description": "Install Plex Media server",
+                                "command": [
+                                     "get_user_continue \"This operation will install Plex Media server.\n\nDo you wish to continue?\" process_input",
```

Comment:
Use prompt here https://github.com/armbian/configng/blob/main/lib/armbian-configng/config.ng.jobs.json#L23 and everywhere else down in the file.

Reply:
Used prompt for all of my menu items.
PR done for these minor tweaks.